### DISCUSSION TITLE
GitHub Desktop NOT WORKING - EADDRNOTAVAIL: address not available 127.0.0.1

### DISCUSSION THREAD

**Discussion by STROUDICOUS** [community/community#110272] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I'm trying to set up a local server using GitHub Desktop to clone my repository to a local dir.

In a browser, 127.0.0.1: 8080 produces a "Site Can't be Accessed" error. However, "localhost" = "It Works!"

Setup details. Mac M1 Sonoma.

<img width="403" alt="image" src="https://github.com/community/community/assets/54401792/06ec9c32-4cd3-412d-96ad-e604bfd291e0">

**Comment by steveward** [comment]
@STROUDICOUS the log file may show some more specific error output. To access the log files go to the file menu in GitHub Desktop and select `Help` > `Show Logs`. The log files are created daily -- please upload a log file as an attachment from a day where you experienced the issue. 

The `EADDRNOTAVAIL` normally indicates something else is using that port. What happens if you ping that IP?
